llmfoundry/models/layers/llama_attention_monkeypatch.py

```diff
@@ -78,6 +78,8 @@ def llama_attention_patch_torch(
     past_key_value: Optional[Tuple[torch.Tensor]] = None,
     output_attentions: bool = False,
     use_cache: bool = False,
+    # Temporary fix for llama2 transformers compatibility, padding_mask will be removed in the next transformers release >4.32.1.
```

Comment:
>=4.34.1 would be the next release, right?

Reply:
oops typo, yeah 4.34.1 is the latest, so anything >4.34.1